corehq/apps/userreports/tasks.py

```diff
@@ -47,7 +47,7 @@ def _build_indicators(indicator_config_id, relevant_ids):
             # save is a noop if the filter doesn't match
             adapter.save(doc)
             redis_client.srem(redis_key, doc.get('_id'))
-        except DataError as e:
+        except Exception as e:
             logging.exception('problem saving document {} to table. {}'.format(doc['_id'], e))
```

Comment:
I mostly wanted to make sure you had a reason for doing it this way which it sounds like you do.